pom.xml

```diff
@@ -34,14 +34,14 @@
 
     <!-- Current target version of m2e, used to deduce some URL at verify and publish.
     This should usually be sync'd with the main feature version -->
-    <m2e.version>1.16.2</m2e.version>
+    <m2e.version>1.17.0</m2e.version>
     <m2e-maven-runtime.version>1.16.0-SNAPSHOT</m2e-maven-runtime.version>
```

Comment:
1.17.0-SNAPSHOT

all versions have been bumped elsewhere, so let's stay consistent

Reply:
Please see comment, this version is used for releng scripts and -SNAPSHOT is not desired in repository URLs (so far).

Reply:
the comment is for L38, which contains SNAPSHOT already

Reply:
In that case, it's not something we want to do either. If we change that, then it means we have to build newer versions of the maven runtime while it actually didn't change at all, leading to distribution of 10MB of redundant content to the millions of users. It's something we now avoid by decoupling runtime version from consuming m2e version.